admin/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.knubisoft</groupId>
+        <artifactId>knubisoft-pirozochki</artifactId>
+        <version>1.0.0</version>
+    </parent>
+    <groupId>com.knubisoft</groupId>
+    <artifactId>admin</artifactId>
+    <version>1.0.0</version>
+    <name>admin</name>
+    <description>admin</description>
+    <properties>
+        <java.version>1.8</java.version>
+        <spring-boot-admin.version>3.0.0-M4</spring-boot-admin.version>
```

Comment:
Not used. U should update spring module versions only using spring boot manager.

Reply:
Remove it